### PR TITLE
Updating the ACL LOG doc in acl-log.md.

### PR DESCRIPTION
**Pull request by roshkhatri**
Updating the doc for the changes made in [redis/redis#11477](https://github.com/redis/redis/pull/11477).

Added description for the ACL LOG fields.

### CODE REVIEW

File: commands/acl-log.md [hunk 1]
```diff
@@ -35,7 +35,26 @@ When called with `RESET`:
...
+   14) "id=3 addr=127.0.0.1:57275 laddr=127.0.0.1:6379 fd=8 name= age=16 idle=0 flags=N db=0 sub=0 psub=0 ssub=0 multi=-1 qbuf=48 qbuf-free=16842 argv-mem=25 multi-mem=0 rbs=1024 rbp=0 obl=0 oll=0 omem=0 tot-mem=18737 events=r cmd=auth user=default redir=-1 resp=2"
+   15) "entry-id"
+   16) (integer) 0
+   17) "timestamp-created"
+   18) (integer) 1675361492408
+   19) "timestamp-last-updated"
+   20) (integer) 1675361492408
 ```
+
+Each LOG is composed of the following fields of the corresponding acl log and their values:
```
Comment (madolson) [inline commands/acl-log.md:49]:
```suggestion
Each log entry is composed of the following fields:
```

Comment (roshkhatri) [reply commands/acl-log.md:49]: will make the necessary changes

File: commands/acl-log.md [hunk 2]
```diff
@@ -35,7 +35,26 @@ When called with `RESET`:
...
+   16) (integer) 0
+   17) "timestamp-created"
+   18) (integer) 1675361492408
+   19) "timestamp-last-updated"
+   20) (integer) 1675361492408
 ```
+
+Each log entry is composed of the following fields:
+
+1. `count`: The number of security events detected within a 60 second periods that are represented by this entry.
```
Comment (madolson) [inline commands/acl-log.md:51]:
```suggestion
1. `count`: The number of security events detected within a 60 second period that are represented by this entry.
```

File: commands/acl-log.md [hunk 3]
```diff
@@ -35,7 +35,26 @@ When called with `RESET`:
...
+2. `reason`: The reason for which the ACL security events were logged. Either `command`, `key`, `channel`, or `auth`.
```
Comment (madolson) [inline commands/acl-log.md:52]:
```suggestion
2. `reason`: The reason that the security events were logged. Either `command`, `key`, `channel`, or `auth`.
```

**Review by madolson — COMMENTED** [review]
Two more nitpicks.

**Pull request merged**